docs/user_guide/assessment.rst

```diff
@@ -1,20 +1,109 @@
-Assessment
-==========
+Introduction
+============
 
-Metrics
--------
+The goal of fairness assessment is to answer the question: Which groups of 
+people may be disproportionately negatively impacted by an AI system and in 
+what ways?
 
-.. currentmodule:: fairlearn.metrics
+The steps of the assesment are as follows:
+1. Identify harms
+2. Identify the groups that might be harmed
+3. Quantify harms
+4. Compare quantified harms across the groups
+
+We next examine these four steps in more detail.
+
+1. Identify harms
+-----------------
+
+For example, in a system for screening job applications, qualified candidates 
+that are automatically rejected experience an allocation harm. In a 
+speech-to-text transcription system, high error rates constitute harm in the 
+quality of service.
```

Comment:
Maybe add a caveat about how systems can lead to multiple harms at the same time. Feel free to re-use language or link to the SciPy tutorial overview slides!
https://github.com/fairlearn/talks/blob/main/2021_scipy_tutorial/overview.pdf

Reply:
I added a caveat at the end of this bullet!